shadow.yaml

```diff
@@ -185,7 +185,7 @@ subpackages:
       - runs: |
           mkdir -p ${{targets.subpkgdir}}/usr/bin
           mv ${{targets.destdir}}/usr/bin/getsubids ${{targets.subpkgdir}}/usr/bin/
-          mv ${{targets.destdir}}/usr/bin/new*idmap ${{targets.subpkgdir}}/usr/bin/
+          cp ${{targets.destdir}}/usr/bin/new*idmap ${{targets.subpkgdir}}/usr/bin/
```

Comment:
Instead of `cp` and have 2 subpkgs that provides same binaries, was wondering if we should have the `just-newuidmap-newgidmap` as a runtime dep here. And omit this line. This package also has tests for those 2 binaries, it does seem like a duplicate. Wdyt?

```suggestion
```

Reply:
I think I get what you are saying, I have made the changes, wdyt @Dentrax